next-env.d.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
+/// <reference path="./.next/types/routes.d.ts" />
```

Comment:
When you rerun `npm run dev` with your current branch code, does this change back to what it was before or does it stay the same?  It seems like this shouldn't be changing with your current code